cli/smartcontract/contract_test.go

```diff
@@ -280,6 +281,11 @@ func TestContractInitAndCompile(t *testing.T) {
 	t.Run("excessive parameters", func(t *testing.T) {
 		e.RunWithError(t, append(cmd, "something")...)
 	})
+	cmdGoModTidy := exec.Command("go", "mod", "tidy")
+	cmdGoModTidy.Dir = ctrPath
+
+	err = cmdGoModTidy.Run()
+	require.NoError(t, err, "Running go mod tidy failed")
```

Comment:
Why do you need this? What is the exact reason of the test failure? Have you managed to reproduce the test failure on the current master locally or in the testing jobs? 

The only dependency this example SC has is `interop` package. This dependency is being replaced by the local package. So the question is whether this issue still relevant, because now minimum required go version is 1.19 for both neo-go and interop.

Reply:
In fact, when you run `go tidy`, it also adds minimum required go version to the `go.mod` file (this is the currently installed go version). So I'd suggest to fix contract go.mod template and add minimum required go version by ourselves:
https://github.com/nspcc-dev/neo-go/blob/71fb759c0d009218437d0048f7ba085a2a74c66f/cli/smartcontract/smart_contract.go#L376-L379

Also may fix the indent while we're here.

Reply:
I managed to reproduce it only with go1.18 locally on Ubuntu with the revision of the master with go1.18 (however interop also required go1.18 at that point, I'm still not sure why we had this problem), so it's not relevant now.  With the current master, it's not reproducible (and with go1.18, it's not runnable).

Reply:
also without go mod tidy i got `error while trying to compile smart contract file: err: exit status 1: stderr: go: updates to go.mod needed; to update it:
	go mod tidy` 
	(go1.18 ubuntu)

Reply:
I think this problem is not relevant anymore with go 1.19 and updated interop deps.